# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/dvd+rwtools/glibc-2.18.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dvd+rw-tools-7.1/growisofs.c.vanilla	2021-02-24 16:44:27.540086741 +0100
+++ dvd+rw-tools-7.1/growisofs.c	2021-02-24 16:44:41.547087463 +0100
@@ -453,6 +453,7 @@
 # include <poll.h>
 # include <sys/resource.h>
 # include <sys/utsname.h>
+# include <sys/sysmacros.h>
 
 # include <errno.h>
 # define set_errno(e)	(errno=(e))
